docs/topics/mta-web-administrator-view.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-web-administration-view_{context}"]
+= User Interface views
+
+The {ProductName} ({ProductShortName}) {WebName} and has two user interfaces with different functionalities:
+
+* Administrator view
+* Developer view
+
+== Administrator view
+The administrator view is used by administrators to create the {ProductShortName} {WebName} instance environment.
+//![](/Tackle2/Views/AdminMenu.png)
+
+=== Credentials
+The credentials management module enables administrators to create and manage credentials to access private repositories.  It enables architects to assign credentials to applications without having to know their contents.
+//![](/Tackle2/Views/CredMain.png)
+
+The credentials page displays the available credentials, *Edit* and *Delete* buttons and the following fields:
+
+* *Name*
+* *Description*
+* *Type*
+** *Source Control*
+** *Username/Password*
+** *Source Private Key/Passphrase*
+** *Maven Settings File*
+** *Proxy*
+* *Type specific information*
+* *Created by*
+
+NOTE: Credential information entered, such as keys and passphrases, is hidden or displayed as [Encrypted].
```

Comment:
This is not true for Maven Settings files upon creation. When the file is uploaded, its contents are displayed in the text box for the user to make tweaks if desired. [Encrypted] only appears on edit for Maven Settings files.

Reply:
I made the requested changes, please review.